environments/prod/main.tf

```diff
@@ -27,6 +27,46 @@ module "cdn_with_s3_bucket" {
   comment        = local.cdn_domain_name
   domain_name    = local.cdn_domain_name
   s3_bucket_name = local.cdn_domain_name
+
+  custom_origins = [
```

Comment:
Should we simplify the input variables for cdn? It is a little bit too verbose I think. 

I guess I reviewed the last PR before you added these 😅

Reply:
You are right! I wanted to be too ... nevermind :D